### PR TITLE
Adding Playlist Page Scrobbling For TrtTurku Connector

### PR DESCRIPTION
**Pull request by TaeZStkyoht**
Playlist page scrobbling has been added. Additionally, Uppercasing problem of initials of  titles of songs with two name which are separated  by '-' character has been resolved.

### CODE REVIEW

**Comment by alexesprit** [comment]
Use `grunt lint` task to see all errors before commiting changes. ;)

**Comment by TaeZStkyoht** [comment]
At least, all checks have passed. How can grunt lint be used?

**Comment by f1u77y** [comment]
@alexesprit what about git hooks?

**Comment by alexesprit** [comment]
@TaeZStkyoht 

```
npm install
npm install -g grunt
grunt lint
```

Of course, Node.js and Node.js package manager are required to be installed.

**Comment by alexesprit** [comment]
@f1u77y what do you mean?

**Comment by f1u77y** [comment]
@alexesprit https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks

**Comment by alexesprit** [comment]
git-hooks are cool stuff, but they are useful for active development. It's enough to run `grunt lint` task before each commit for a start.

File: connectors/v2/trtturku.js [hunk 1]
```diff
@@ -35,3 +45,35 @@ Connector.currentTimeSelector = '.jwelapsed';
...
+}
+
+function setupPlaylistPlayer() {
+	Connector.getArtist = function () {
+		return $('.jwitem.active .jwtitle').text().split(' - ')[0];
+	};
+	
+	Connector.getTrack = function () {
+		return $('.jwitem.active .jwtitle').text().split(' - ')[1];
+	};
```
Comment (alexesprit) [inline connectors/v2/trtturku.js]: This code can be simplified by using 'Connector.artistTrackSelector' field:
```js
Connector.artistTrackSelector = '.jwitem.active .jwtitle';
```
The core will perform split by itself.

**Comment by TaeZStkyoht** [comment]
I guess there is no user mechanism on this site and there is no specific playlist URL. But I guess I found a trick. Browse the link http://www.trtturku.net/calma-listem/ and add this to cookie: document.cookie = "playlist_string=5781%2C2028%2C10546%2C"

File: connectors/v2/trtturku.js (hunk 1 above)
Comment (TaeZStkyoht) [inline connectors/v2/trtturku.js]: I applied.

**Pull request merged**